submissions/onyinyeChukwuka/css/sign-up.html

```diff
@@ -0,0 +1,99 @@
+                    <form>
+                        <label for="firstname">First Name</label><br>
+                        <input type="text" class="texts" id="firstname" name="firstname"><br><br>
+                        <label for="lastname">Last Name</label><br>
+                        <input type="text" class="texts" id="lastname" name="lastname"><br><br>
+                        <label for="othernames">Other Names</label><br>
+                        <input type="text" class="texts" id="othernames" name="othernames"><br><br>
+                        <label for="phonenumber">Phone Number</label><br>
+                        <input type="text" class="texts" id="phonenumber" name="phonenumber"><br><br>
+                        <label for="email">Email Address </label><br>
+                        <input type="text" class="texts" id="email" name="email"><br><br>
+                        <label for="password">Password</label><br>
+                        <input type="text" class="texts" id="password" name="password"><br><br>
+                        <label for="cpassword">Confirm Password</label><br>
+                        <input type="text" class="texts" id="cpassword" name="cpassword"><br><br><br>
+                
+                        <div class="confirm">
+                            <div>
+                                <input type="checkbox" id="terms" name="terms">
+                                <label for="terms"> I agree to the Terms and Conditions</label><br><br>
+                            </div>
+                            <button type="button" id="signupbutton">SIGN UP</button>
+                        </div>
+                    </form>
+                </div>
+            </div>
+            <footer class="footer">
+                <div class="footer-wrapper">
+                    <div class="about">
+                        <p> Contact Us </p>
+                        <p> About Us </p>
+                        <p> Terms and Conditions </p>
+                    </div>
+        
+                    <div class="socials">
+                        <div class="newsletter">
+                            <label for="email">Sign up for our Newsletter</label><br><br>
+                            <div class="newsletter-input">
+                                <input type="text" id="newslettter" name="email" placeholder="Enter your email address">
+                                <button type="button" id="submitbutton">SUBMIT</button>
+                            </div>
+        
+                        </div>
+        
+                        <div class="connect">
+                            <p> CONNECT WITH US</p>
+                            <div class="images">
+                                <img src="../html/images/logos_facebook.png" alt="facebook_logo" width="45px" height="41px">
+                                <img src="../html/images/twitter.png" alt="twitter_logo" width="48px" height="43px">
```

Comment:
Remove the `width` and `height` here. Style the images in the css file. @Onyinye101